### PR TITLE
Enable visbuilder by default

### LINKED ISSUE #2672: [Vis Builder] Enabled by default
Tracking issue to set VisBuilder to be enabled by default in the config.

### PR DESCRIPTION
**Pull request by ashwin-pc**
Signed-off-by: Ashwin P Chandran <ashwinpc@amazon.com>

### Description
Enables VisBuilder by default 
### Issues Resolved
#2672
 
### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
  - [x] `yarn test:jest_integration`
  - [x] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [x] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by AMoo-Miki — APPROVED** [review]

**Review by seanneumann — APPROVED** [review]
Nice! Congrats to moving to the default on stage. :)

**Review by bandinib-amzn — APPROVED** [review]

File: src/plugins/vis_builder/server/index.ts [hunk 1]
```diff
@@ -17,8 +17,5 @@ export function plugin(initializerContext: PluginInitializerContext) {
 export { VisBuilderPluginSetup, VisBuilderPluginStart } from './types';
 
 export const config: PluginConfigDescriptor<ConfigSchema> = {
-  exposeToBrowser: {
-    enabled: true,
-  },
```
Comment (joshuarrrr) [inline src/plugins/vis_builder/server/index.ts:22]: Why do we no longer need this?

**Review by joshuarrrr — APPROVED** [review]
With this change, I think there are some dockerfile options we can also remove...

File: src/plugins/vis_builder/server/index.ts (hunk 1 above)
Comment (manasvinibs) [reply src/plugins/vis_builder/server/index.ts:22]: Yeah same question. My understanding was we can use the config schema in the `plugin.ts` class constructor, inside public folder of the plugin only when we set `exposeToBrowser` on enabled property to true. Without this settings, is it possible to disable the visbuilder during plugin registration time?

Comment (kavilla) [reply src/plugins/vis_builder/server/index.ts:22]: @manasvinibs I believe this setting is only used at registration and before browser rendering. So I believe this probably was never used unless there was custom logic in the public folder to check this config

**Comment by kavilla** [comment]
@ashwin-pc @ananzh it's still marked experimental though right? But I know @ananzh's table was basing on if vis builder was enabled or not. Is this still okay?

**Comment by AMoo-Miki** [comment]
Manual 2.4 backport: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2773

**Comment by ananzh** [comment]
> 

yeah table vis is only used in vis builder. with this PR, visBuilder will be enabled as a default vis. When cx click visBuilder, they can choose table visualization. But if just click `data table` in visualization page, they will use the old table vis.

**Pull request merged**